anomalib/models/draem/torch_model.py

```diff
@@ -0,0 +1,495 @@
+"""PyTorch model for the DRAEM model implementation."""
+
+# Original Code
+# Copyright (c) 2021 VitjanZ
+# https://github.com/VitjanZ/DRAEM.
+# SPDX-License-Identifier: MIT
+#
+# Modified
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# pylint: disable=invalid-name
```

Comment:
Why is this added here?

Reply:
The torch model uses some single letter variable names, e.g. `def forward(x):`. Similar to FastFlow, we could refactor this later.

Reply:
Can't we just replace it with batch? Otherwise it might be too many todos for the future